dev/tasks/conda-recipes/azure.win.yml

```diff
@@ -34,11 +34,10 @@ jobs:
 
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.6 conda-build conda conda-forge::conda-forge-ci-setup=2' # Optional
```

Comment:
Can you switch this back to 2 for now?

Reply:
It didn't affect the build error, but at least it's up to date with the upstream feedstock.